assets/js/src/settings/index.tsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import jQuery from 'jquery';
+import 'parsleyjs';
```

Comment:
This is because the Modal uses `parsley` to validate the input. Please note that this is the webpack entry point for the settings page, it doesn't use the common entry point that already includes jQuery and parsley.